[export] Expand coverage to more copied sym ops for unflattener.

zhxchen17 opened the pull request:
Test Plan:
buck2 test 'fbcode//mode/opt' fbcode//torchrec/ir/tests:test_serializer -- --run-disabled

```
File changed: fbcode//caffe2/torch/export/unflatten.py
Buck UI: https://www.internalfb.com/buck2/2e0377e7-e2b6-4bd0-8133-a787245165a0
Test UI: https://www.internalfb.com/intern/testinfra/testrun/5066549824883887
Network: Up: 0B  Down: 0B
Jobs completed: 16. Time elapsed: 10.2s.
Tests finished: Pass 6. Fail 0. Fatal 0. Skip 0. Build failure 0
```

Differential Revision: D62190172

pianpwk commented on torch/export/unflatten.py:
```diff
@@ -866,7 +866,14 @@ def remap_input(self, x):
             return self.node_to_placeholder[x]
         elif x.op == "call_function" and (
-            x.target == torch.ops.aten.sym_size.int
+            x.target in (
```
I think this makes sense with CSE, but just wondering how you ended up with this list?

zhxchen17 replied:
I just collected all the ops required from torchrec :) I made a list because previously this will include every op in the universe (including the embedding lookup custom ops), which will make debugging pretty hard compared to failing fast. 

I agree that in the future we should add a better protection for submodules.

pianpwk replied:
I see, I'm a bit surprised with view & diff showing up here? sym_size & item make sense because they're SymInt nodes that get deduped by CSE, and need to be recopied. But those 2 are just tensor compute, which CSE doesn't touch?

Do you know how things are failing there?

zhxchen17 committed:
[export] Expand coverage to more copied sym ops for unflattener. (#135119)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/135119

Test Plan:
buck2 test 'fbcode//mode/opt' fbcode//torchrec/ir/tests:test_serializer -- --run-disabled

```
File changed: fbcode//caffe2/torch/export/unflatten.py
Buck UI: https://www.internalfb.com/buck2/2e0377e7-e2b6-4bd0-8133-a787245165a0
Test UI: https://www.internalfb.com/intern/testinfra/testrun/5066549824883887
Network: Up: 0B  Down: 0B
Jobs completed: 16. Time elapsed: 10.2s.
Tests finished: Pass 6. Fail 0. Fatal 0. Skip 0. Build failure 0
```

Reviewed By: PaulZhang12

Differential Revision: D62190172
